controllers/drplacementcontrol.go

```diff
@@ -606,6 +578,11 @@ func (d *DRPCInstance) ensureCleanupAndVolSyncReplicationSetup(srcCluster string
 func (d *DRPCInstance) quiesceAndRunFinalSync(homeCluster string) (bool, error) {
 	const done = true
 
+	d.setDRState(rmn.Relocating)
```

Comment:
I don't think we should have this block here. Quiesceing the app doesn't mean that we are relocating. I think `Initiating` as it is set just before this call is good enough.  `Relocating` means we started to mess up with the VRGs

Reply:
The point at which we change user placement to `nil` is when the application goes offline (or starts to go offline) for relocation. I was hence thinking we can mark relocating when we do the same, i.e just before line 597 below (before calling `clearUserPlacementRuleStatus`). We can move this block around there, thoughts?

Reply:
Line 596 looks like a good place for it.